docker-compose.yml

```diff
@@ -48,6 +48,35 @@ services:
       - 6001:6000
     depends_on:
       - yaba_api
+  
+  yaba_visualization:
+    build: ./visualization
+    networks:
+      - bety
+    volumes:
+      - './visualization:/code'
+      - '/code/node_modules'
+    ports:
+      - 80:8080
```

Comment:
Will these ports conflict with Pecan? https://github.com/PecanProject/pecan/blob/3b0cb56bba1eb90d1f7bbe3b0da3510f681519ba/docker-compose.yml#L70 and https://github.com/PecanProject/pecan/blob/3b0cb56bba1eb90d1f7bbe3b0da3510f681519ba/docker-compose.yml#L153

Reply:
Yeah! it may interfere with pecan and increase traffic on the port, so should I change the ports?

Reply:
@im-prakher It would appear that a change is needed. You should check in with the Pecan group to see what they recommend changing them to

Reply:
Okay, Thanks!